src/collectors/jolokia/jolokia.py

```diff
@@ -60,6 +68,9 @@ def get_default_config_help(self):
             "stats. If not provided, all stats will be collected",
             'host': 'Hostname',
             'port': 'Port',
+            'instances': "Jolokia addresses, comma separated, syntax:"
+            + " nick1@host:port, nick2@host:port"
```

Comment:
Instead of concatenating the string literal you can do:
```python
'instances': ("Jolokia addresses, comma separated, syntax:"
              " nick1@host:port, nick2@host:port")
```